src/parsing/xpath.pegjs

```diff
@@ -1196,7 +1196,7 @@ SingleType
  = typeName:SimpleTypeName optional:"?"? {return optional ? ["singleType", ["atomicType"].concat(typeName), ["optional"]] : ["singleType", ["atomicType"].concat(typeName)]}
 
 // 183
-TypeDeclaration
+ValueType
```

Comment:
We removed the original TypeDeclaration since it stored a type and a multiplicity which ValueType also did. ValueType is now renamed to SequenceType, so should we change it back to SequencType, or add TypeDeclaration back in?

Reply:
this is the parser, the TypeDeclaration string there is just something from the grammar of XPath/XQuery

Reply:
Ooh my bad thought it was linked somehow